### PR TITLE
[4.2.0] Update email-connector-config.md

### PR DESCRIPTION
**Pull request by DinithiDiaz**
## Purpose
$subject

Fixes https://github.com/wso2/docs-mi/issues/65.

### CODE REVIEW

**Comment by DinithiDiaz** [comment]
Updated UI:
email-4 2

**Comment by chanikag** [comment]
InlineImages is a parameter inside send operation.

**Comment by DinithiDiaz** [comment]
Updated UI:


https://github.com/user-attachments/assets/b9c52548-0242-4975-b8d7-f466ff1d1bea

File: en/docs/reference/connectors/email-connector/email-connector-config.md [hunk 1]
```diff
@@ -418,6 +418,59 @@ The following operations allow you to work with the Email Connector. Click an op
...
+                        "fileName": "image2.jpeg",
+                        "base64Content": "/9j/4AAQSkZJRgABAQEBLAEsAAD/4QBbRXhp...."
+                    }
+                ],
+                "contentType": "text/html"
+            }
+            ```
+            </div>  
+            </td>
+            <td></td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/email-connector/email-connector-config.md:472]:
```suggestion
            <td>Optional</td>
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**